### PR TITLE
Solved issues in Barometer instrument

### LINKED ISSUE #1364: Problems in bottom sheet of barometer instrument
**Actual Behaviour**

The bottom sheet guide isn't opening after closing it when the app is started for the first time. Also, the indicator in the barometer overlaps with the bottom sheet guide.

**Expected Behaviour**

The bottom sheet should slide smoothly without any issues

**Steps to reproduce it**

Go to Barometer instrument -> Analyze

**LogCat for the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by harsh-2711**
Fixes #1364 

**Changes**: 
1. Solved the issue of indicator overlapping in Barometer instrument
2. Changed the indicator in Barometer instrument to make it uniform
3. Made Barometer start working when pressure sensor is present in the mobile
4. Solved the issue of setting max and min in the statistics row

**Screenshot/s for the changes**: 
screen shot 2018-10-01 at 12 38 22

screen shot 2018-10-01 at 15 27 54

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/2433514/app-debug.apk.zip)

### CODE REVIEW

File: app/src/main/res/layout/activity_barometer.xml [hunk 1]
```diff
@@ -20,4 +20,289 @@
...
+                    app:sv_speedTextColor="@color/black"
+                    app:sv_speedTextPosition="BOTTOM_CENTER"
+                    app:sv_speedometerMode="NORMAL"
+                    app:sv_unit=" atm"
+                    app:sv_unitTextColor="@color/black"
+                    app:sv_textSize="@dimen/text_size"
+                    app:sv_unitTextSize="@dimen/text_size" />
+
+
+                <!-- <com.github.anastr.speedviewlib.TubeSpeedometer
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_barometer.xml]: Remove this commented block

**Comment by harsh-2711** [comment]
@CloudyPadmal Changes done. I think @mariobehling told in one of our meetings to make all the gauges uniform.

**Comment by CloudyPadmal** [comment]
Yes uniform, but which one to keep red or blue one?

**Comment by harsh-2711** [comment]
@CloudyPadmal @abhinavraj23 @Avjeet Please review.

**Comment by abhinavraj23** [comment]
@Avjeet @CloudyPadmal Please review

**Pull request merged**